linker/Tests/Mono.Linker.Tests.Cases.Expectations/Metadata/StripResourcesAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Mono.Linker.Tests.Cases.Expectations.Metadata {
+	public sealed class StripResourcesAttribute : BaseMetadataAttribute {
```

Comment:
It's fine that you added this, don't worry about removing it.  But FYI for the future, for simple command line flags that communicate their meaning you can use the general purpose `SetupLinkerArgumentAttribute` 

Ex: `[SetupLinkerArgument ("--strip-security", "true")]`

What `SetupLinkerArgumentAttribute` shouldn't be used for is the command line flags that communicate nothing to someone looking at a test.  For example, don't use it for:  `[SetupLinkerArgument ("-r", "library.dll")].

Reply:
@mrvoorhe I'm merging it as we are not consistent with other command line options either though it'd be nice to clean this up